server/storage/mvcc/kvstore_compaction_test.go

```diff
@@ -110,7 +110,10 @@ func TestScheduleCompaction(t *testing.T) {
 func TestCompactAllAndRestore(t *testing.T) {
 	b, tmpPath := betesting.NewDefaultTmpBackend(t)
 	s0 := NewStore(zaptest.NewLogger(t), b, &lease.FakeLessor{}, StoreConfig{})
-	defer os.Remove(tmpPath)
+	defer func() {
+		b.Close()
+		os.Remove(tmpPath)
```

Comment:
Isn't the `tmpPath` automatically cleaned up? 


see https://pkg.go.dev/testing#T.TempDir 
```
TempDir returns a temporary directory for the test to use. The directory is automatically removed
by Cleanup when the test and all its subtests complete. Each subsequent call to t.TempDir returns 
a unique directory; if the directory creation fails, TempDir terminates the test by calling Fatal. 
```

Reply:
Good point! Let me see how to change this. Update it later.

Reply:
Updated. Please take a look. Thanks!